From 41e861f9886e4c1e302829ab0064a394f350eb58 Mon Sep 17 00:00:00 2001
From: Peter Swain <peter.swain@marvell.com>
Date: Tue, 3 Jul 2018 15:28:33 -0700
Subject: [PATCH 167/767] gpio: octeontx2: extend PIN_SEL to cover cn96xx

commit 4310cb48d3cbeb45653d64891dbed98348c88e03 from
git@git.assembla.com:cavium/WindRiver.linux.git

Expand GPIO_BIT_CFG_PIN_SEL_MASK to cover the new bit added at top.
As with previous expansions of this field, it extends into a space
that was RAZ in all previous chips, so no version-checks are needed.

Change-Id: I6453dcdee354aea201772236204d1642f015b82a
Signed-off-by: Peter Swain <peter.swain@marvell`.com>
Reviewed-on: https://sj1git1.cavium.com/8762
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/gpio/gpio-thunderx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpio/gpio-thunderx.c b/drivers/gpio/gpio-thunderx.c
index 715371b5102a..6bc57b413d0e 100644
--- a/drivers/gpio/gpio-thunderx.c
+++ b/drivers/gpio/gpio-thunderx.c
@@ -31,7 +31,7 @@
 #define  GPIO_BIT_CFG_FIL_CNT_SHIFT	4
 #define  GPIO_BIT_CFG_FIL_SEL_SHIFT	8
 #define  GPIO_BIT_CFG_TX_OD		BIT(12)
-#define  GPIO_BIT_CFG_PIN_SEL_MASK	GENMASK(25, 16)
+#define  GPIO_BIT_CFG_PIN_SEL_MASK	GENMASK(26, 16)
 #define GPIO_INTR	0x800
 #define  GPIO_INTR_INTR			BIT(0)
 #define  GPIO_INTR_INTR_W1S		BIT(1)
-- 
2.31.1

